packages/calypso-build/bin/cb.js

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
```

Comment:
Thinking of maybe renaming the file to `calypso-build.js`, or just `build.js` so it's a bit more evident what it does. (I do want to keep the `cb` bin alias though.)

Reply:
Not that it matters that much tho, on second thought :man_shrugging:

Reply:
> so it's a bit more evident what it does.

A comment on top of the file might help more. Having the bin alias and filename called the same might be less confusing.